### PR TITLE
[SIG-2506] map style improvements

### PR DESCRIPTION
**Pull request by aditudorache**
This PR
- updates amsterdam-styled-components package
- improves the styling in the map component as specified in the design

### CODE REVIEW

File: src/components/Map/index.js [hunk 1]
```diff
@@ -23,13 +31,7 @@ const Map = ({ className, mapOptions, hasZoomControls, canBeDragged, children, e
...
 
   return (
-    <MapComponent
-      className={className}
-      data-testid="map-base"
-      options={options}
-      events={events}
-      setInstance={setInstance}
-    >
+    <StyledMap className={className} data-testid="map-base" options={options} events={events} setInstance={setInstance}>
```
Comment (janjaap) [inline src/components/Map/index.js:34]: This line is longer than the linter should allow

File: src/components/Map/index.js [hunk 2]
```diff
@@ -9,6 +9,14 @@ const StyledViewerContainer = styled(ViewerContainer)`
   z-index: 400; // this elevation ensures that this container comes on top of the internal leaflet components
 `;
 
+const StyledMap = styled(MapComponent)`
+  cursor: default;
+
+  &.leaflet-drag-target {
+    cursor: all-scroll;
+  }
```
Comment (janjaap) [inline src/components/Map/index.js:17]: Wouldn't `grab` and `grabbing` make more sense in this case?

Comment (aditudorache) [reply src/components/Map/index.js:17]: These are the requests of the designer.

File: src/components/Map/index.js (hunk 1 above)
Comment (aditudorache) [reply src/components/Map/index.js:34]: No problem here. printWidth is 120 in .prettierrc

**Pull request merged**